chaoscenter/subscriber/pkg/k8s/operations.go

```diff
@@ -185,8 +184,9 @@ func (k8s *k8sSubscriber) AgentRegister(infraData map[string]string) (bool, erro
 	logrus.Info(InfraConfigName + " has been updated")
 
 	newSecretData := map[string]string{
-		"ACCESS_KEY": infraData["ACCESS_KEY"],
-		"INFRA_ID":   infraData["INFRA_ID"],
+		"ACCESS_KEY":        infraData["ACCESS_KEY"],
+		"INFRA_ID":          infraData["INFRA_ID"],
+		"CUSTOM_TLS_SECRET": infraData["CUSTOM_TLS_SECRET"],
```

Comment:
@Saranya-jena  - where are we setting the value CUSTOM_TLS_SECRET? In system env or from the ui

Reply:
As far as I know we are setting the the CUSTOM_TLS_SECRET from the system env